extensions/markdown-math/syntaxes/md-math-block.tmLanguage.json

```diff
@@ -13,7 +13,7 @@
 		"double_dollar_math_block": {
 			"name": "markup.math.block.markdown",
 			"contentName": "meta.embedded.math.markdown",
-			"begin": "(?<=^\\s*)(\\${2})(?![^$]*\\${2})",
+			"begin": "(?:.*?)(\\${2})(?![^$]*\\${2})",
```

Comment:
Should probably use `(?<=^|\\s+)` instead
as `(?:.*?)` tries to capture everything